### PR TITLE
Update a bunch of product ids

### PR DESCRIPTION
**Pull request by kodawah**

### CODE REVIEW

File: data/products/DSK.yaml [hunk 1]
```diff
@@ -5,20 +5,23 @@ products:
     identifiers:
       cardtraderId: '292182'
       mcmId: '776445'
+      tcgplayerProductId: '557249'
     release_date: '2024-07-02'
```
Comment (axxroytovu) [inline data/products/DSK.yaml:9]: While we're in the file we should probably fix the release date for these products.

Comment (kodawah) [reply data/products/DSK.yaml:9]: why, is it wrong?

Comment (axxroytovu) [reply data/products/DSK.yaml:9]: The release date for Duskmourn is 9/27 but these products say 7/2. It was probably copied over automatically from one of the providers that use "the date we put it in the system" as the "release date"

Comment (kodawah) [reply data/products/DSK.yaml]: ok updated

**Review by axxroytovu — APPROVED** [review]

**Pull request merged**